src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerMessage = 'I am not married';
+
+  if (person.isMarried) {
+    const relationship = person.sex === 'm' ? 'wife' : 'husband';
+
+    partnerMessage =
+      person.sex === 'm'
+        ? `${person.partnerName} is my ${relationship}`
+        : `${person.partnerName} is my ${relationship}`;
```

Comment:
```suggestion
    partnerMessage = `${person.partnerName} is my ${relationship}`;
```
In both cases you set the same, what is the point of doing it?